src/main/resources/application.properties

```
@@ -0,0 +1,10 @@
+spring.datasource.url=jdbc:h2:file:c:\\Users\\vlykh\
+  \\IdeaProjects\\bankoflykhvar\\src\\main\\resources\\data\\data;AUTO_SERVER=true
+spring.h2.console.enabled=true
```

```suggestion
spring.h2.console.enabled=true
spring.h2.console.path=/h2
```
by which path is the console available at the moment?